tools/quantize/quantize.cpp

```diff
@@ -643,6 +644,11 @@ int main(int argc, char ** argv) {
         return 1;
     }
 
+    if (std::filesystem::exists(fname_out) && std::filesystem::equivalent(fname_inp, fname_out)) {
+        fprintf(stderr, "%s: error: input and output files are the same: '%s'\n", __func__, fname_inp.c_str());
+        return 1;
+    }
+
```

Comment:
Perhaps `std::filesystem::exists` call can be avoided here:
```suggestion
    if (std::error_code ec; fs::equivalent(fname_inp, fname_out, ec)) {
        fprintf(stderr, "%s: error: input and output files are the same: '%s'\n", __func__, fname_inp.c_str());
        return 1;
    }
````